server/models/Comment.ts

```diff
@@ -51,6 +56,9 @@ class Comment extends ParanoidModel<
   @Column(DataType.JSONB)
   data: ProsemirrorData;
 
+  @Column(DataType.JSONB)
+  reactions: Reaction[] | null;
```

Comment:
Please note the structure has changed. The array structure makes it simple to construct mobx observables in the frontend `Store` (follows `collection.documentStructure`).
**Proposed**
```
{
  "emoji_a": ["user_id_a", "user_id_b"],
  "emoji_b": ["user_id_b"]
}
```
**Current**
```
[
  {
    "emoji": "emoji_a",
    "userIds": ["user_id_a", "user_id_b"]
  },
  {
    "emoji": "emoji_b",
    "userIds": ["user_id_b"]
  }
]
```